RuntimePerformanceTests/Benchmarks/LanguageCoverage/Benchmark.swift

```diff
@@ -43,7 +43,7 @@ let benchmarks: @Sendable () -> Void = {
         },
         reverse: { benchmark in
             for _ in benchmark.scaledIterations {
-                let thing = gradient(at: 2, of: oneOperation)
+                let thing = gradient(at: 2, of: { v in oneOperation(a: v) })
```

Comment:
Unfortunate we have to add this fix in here. Would you mind adding a reproducer to swift-differentiation-testing?

Reply:
https://github.com/differentiable-swift/swift-differentiation-testing/pull/16

Haven't finished that reproducer yet but yeah